cmd/security-secretstore-read/main.go

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright 2020 Redis Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
+
+	"github.com/edgexfoundry/edgex-go/internal/security/secretread"
+)
+
+func main() {
+	lc := logger.NewClient(secretread.ServiceKey, false, "logs/security-secretstore-read.log", models.ErrorLog)
```

Comment:
@lenny-intel: Lenny, any comments on what we should be doing here WRT logging?

@andresrinivasan: Don't really like the relative logging path hard coded in.  Will be harder to make the container root FS read-only later.

Reply:
Agreed this is weak sauce. I am committed to properly integrate with go-mod-bootstrap but I ran out of time as I was getting this to work in time for code freeze.

Reply:
Noted the lack of integration with go-mod-bootstrap.  Will not make further comments on it in this PR.

Reply:
Tracking go-mod-bootstrap issue via #2503